src/test/java/uk/gov/hmcts/sptribs/common/notification/CaseIssuedNotificationTest.java

```diff
@@ -162,12 +191,17 @@ void shouldNotifyRespondentOfCaseIssuedWithEmailWithAttachments() {
 
         //Then
         verify(notificationService).sendEmail(any(NotificationRequest.class));
+
```

Comment:
Missing verify for buildEmailNotificationRequest

Reply:
My bad missed that one, Thanks